creatio/style.css

```diff
@@ -7,11 +7,11 @@ Description: Creatio is a simple, minimal theme that supports full-site editing
 Requires at least: 6.0
 Tested up to: 6.2.3
```

Comment:
dotOrg suggest to have only upto minor version. Update it to `6.2`.

Also, theme uses fluid typography. Means the required min version should be `6.1`
Update the same in `readme.txt`

Reply:
Done!